tests/components/samsungtv/test_media_player.py

```diff
@@ -77,28 +77,30 @@ def tearDown(self):
 
     @MockDependency("samsungctl")
     @MockDependency("wakeonlan")
```

Comment:
I'm not sure the decorator will work on coroutine test functions. If it doesn't work, break the decorator off and use the context manager version inside pytest fixtures instead. `yield` inside the context manager to keep the mock in place during the test.

```py
@pytest.fixture(name="samsung")
def mock_samsung_fixture():
    with MockDependency(...):
        yield
```

Reply:
@MartinHjelmare Do you mean like this? https://github.com/home-assistant/home-assistant/pull/27492/commits/821ff78d03dec416df22d019e80158bd9a3905de